dapr-spring-core/src/main/java/io/diagrid/spring/core/keyvalue/DaprKeyValueAdapter.java

```diff
@@ -112,27 +135,27 @@ public void clear() {
     public <T> Iterable<T> find(KeyValueQuery<?> query, String keyspace, Class<T> type) {
         DaprMetadata metadata = daprClient.getMetadata().block();
         List<ComponentMetadata> components = metadata.getComponents();
-        String statestoreTypeAndVersion = findStatestoreTypeAndVersion(statestoreName, components);
+        String stateStoreTypeAndVersion = findStateStoreTypeAndVersion(stateStoreName, components);
 
-        if (statestoreTypeAndVersion.isEmpty()) {
+        if (stateStoreTypeAndVersion.isEmpty()) {
             return Collections.emptyList();
         }
 
-        if (!statestoreTypeAndVersion.equals("state.postgresql-v1")) {
+        if (!stateStoreTypeAndVersion.equals("state.postgresql-v1")) {
```

Comment:
@artur-ciocanu we need to make `state.postgresql` and v1 constants here probably.. so then we can combine them.. wdyt?

Reply:
If you don't mind I want to leave the `find` as is for now to focus on other operations. Then I want to spend more time on `find` and figure what should be extracted as constants and how to better handle the filtering.